app/docs/enterprise/0.31-x/developer-portal/authentication.md

```diff
@@ -25,34 +25,43 @@ title: Authenticating the Developer Portal
 ## Enable Authentication
 
 
-First, create an API to proxy requests to the Public Dev Portal API:
+First, create a Service and corresponding Route to proxy requests to the Public Dev Portal API:
```

Comment:
elsehwere you refer to the "Public Dev Portal Files API" (what a mouthful!) - this mention should use the same terminology for clarity.